src/view_page.cpp

```diff
@@ -1745,6 +1745,15 @@ void View::DrawTextChildren(DeviceContext *dc, Object *parent, TextDrawingParams
         }
     }
 
+    if ((!parent->GetChildCount()) && (m_doc->GetType() == Facs) && (m_doc->m_notationType == NOTATIONTYPE_neume)) {
+        // add child to empty syl in facsimile mode for neume notation
+        Text *text = new Text();
+        std::u32string str = U"";
+
+        text->SetText(str);
+        parent->AddChild(text);
+    }
+
```

Comment:
Adding some content to the data when rendering the page is very weird. Can't this be done when loading the file? It would be slightly better.